.travis/debian7/Dockerfile

```diff
@@ -10,7 +10,9 @@ RUN echo 'deb http://archive.debian.org/debian wheezy-backports main contrib' >
 
 RUN echo "Acquire::Check-Valid-Until false;" > /etc/apt/apt.conf
 
-RUN apt-get update && apt-get install -y apt-utils
+RUN apt-get update
```

Comment:
I understand that these images are thrown away after the CI job is completed, but this change goes directly against official docker "best practices". Be careful when applying it to other images.

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Thanks for the hint.